/Library/Developer/CommandLineTools/usr/bin/python3: can't open file '/Users/air/ea_assistant/mcp_server/interface/sync_loops_to_markdown.py': [Errno 2] No such file or directory
/Library/Developer/CommandLineTools/usr/bin/python3: can't open file '/Users/air/ea_assistant/mcp_server/interface/sync_loops_to_markdown.py': [Errno 2] No such file or directory
/Library/Developer/CommandLineTools/usr/bin/python3: can't open file '/Users/air/ea_assistant/mcp_server/interface/sync_loops_to_markdown.py': [Errno 2] No such file or directory
/Library/Developer/CommandLineTools/usr/bin/python3: can't open file '/Users/air/ea_assistant/mcp_server/interface/sync_loops_to_markdown.py': [Errno 2] No such file or directory
/Library/Developer/CommandLineTools/usr/bin/python3: can't open file '/Users/air/ea_assistant/mcp_server/interface/sync_loops_to_markdown.py': [Errno 2] No such file or directory
/Library/Developer/CommandLineTools/usr/bin/python3: can't open file '/Users/air/ea_assistant/mcp_server/interface/sync_loops_to_markdown.py': [Errno 2] No such file or directory
